back/spec/services/local_project_copy_service_spec.rb

```diff
@@ -312,4 +312,241 @@
       end
     end
   end
+
+  describe 'when source project has associated content builder layout' do
+    let!(:layout) { create(:layout, code: 'project_description') }
+
+    it 'copies content builder layout' do
+      copied_project = service.copy(layout.content_buildable)
+
+      expect(copied_project.content_builder_layouts.first
+        .as_json(except: %i[id content_buildable_id updated_at created_at]))
+        .to eq(layout.as_json(except: %i[id content_buildable_id updated_at created_at]))
+    end
+
+    it 'copies associated layout images as new images, associated with copied layout' do
+      image1 = create(:layout_image)
+      image2 = create(:layout_image)
+
+      craftjs = {
```

Comment:
Although this very long json object could make the test difficult to read, I feel that most people use an IDE where such objects can be collapsed, giving a clearer view:

<img width="918" alt="Screenshot 2023-02-01 at 16 38 22" src="https://user-images.githubusercontent.com/3944042/216106009-f5335f1b-85ef-47d0-aed5-280652c2a3b1.png">